### PR TITLE
Fix bundle and mech melee attack animation

### LINKED ISSUE #17037: Plasma amount in the air check doesn't work properly
```
Results:
Pressure: 102 kPa
Nitrogen: 78.86 %
Oxygen: 21.12 %
CO2: 0.01 %
Plasma: 0.01 %
Temperature: 22 °C
```
`safe_tox_max` is set to 0.05 and that much plasma is dealing damage anyway.

Using a flamethrower once in a hallway makes it inhabitable without internals for the rest of the round. And it's nearly impossible to get rid of that little plasma.

### PR DESCRIPTION
**Pull request by phil235**
- <b>Add an attack animation for mech melee attack.</b>
- Fixes mech action button icon not updating whent the button is activated.
- Fixes internal tank action button icon not updating to off when we adjusted our masks out.
- Fixes mime/clown mask not updating their action button icon when adjusting. Fixes clown mask not updating
  its on mob icon if adjusted when worn.
- Fixes typo in the human threshold for toxin in the air. Fixes  #17037
- Fixes changing the thermomachine circuitboard settings not giving any feedback.

### CODE REVIEW

File: code/game/machinery/constructable_frame.dm [hunk 1]
```diff
@@ -377,12 +377,16 @@ to destroy them and players will be able to make replacements.
...
+		var/new_setting = "Heater"
+		playsound(src.loc, 'sound/items/Screwdriver.ogg', 50, 1)
 		if(build_path == initial(heater.build_path))
 			newtype = freezer
+			new_setting = "Freezer"
 		else
 			newtype = heater
 		name = initial(newtype.name)
 		build_path = initial(newtype.build_path)
+		user << "<span class='notice'>You change the circuitboard setting to \"[new_setting]\".</span>"
```
Comment (RemieRichards) [inline code/game/machinery/constructable_frame.dm]: or just ```[Capitalize(newtype)]```

File: code/game/mecha/mecha.dm [hunk 2]
```diff
@@ -1021,41 +1021,34 @@ var/year_integer = text2num(year) // = 2013???
...
-	cycle_action.Grant(user)
-
-	lights_action.chassis = src
-	lights_action.Grant(user)
-
-	stats_action.chassis = src
-	stats_action.Grant(user)
-
-	strafing.chassis = src
-	strafing.Grant(user)
```
Comment (phil235) [inline code/game/mecha/mecha.dm]: removed some lines by giving the chassis assignment code to action/innate/mecha/Grant()

File: code/game/mecha/mecha.dm [hunk 3]
```diff
@@ -1175,12 +1171,11 @@ var/year_integer = text2num(year) // = 2013???
 /datum/action/innate/mecha/mech_toggle_thrusters/Activate()
 	if(!owner || !chassis || chassis.occupant != owner)
 		return
-	var/obj/mecha/M = chassis
```
Comment (phil235) [inline code/game/mecha/mecha.dm]: The var used to be necessary when only certain mech type had access to certain mech actions, it's now useless.

File: code/game/objects/structures/tables_racks.dm [hunk 4]
```diff
@@ -416,8 +416,8 @@
 
 
 /obj/structure/rack/mech_melee_attack(obj/mecha/M)
-	visible_message("<span class='danger'>[M.name] smashes [src] apart!</span>")
-	rack_destroy()
+	if(..())
+		take_damage(M.force*2)
```
Comment (phil235) [inline code/game/objects/structures/tables_racks.dm]: so we get a sound when hitting the rack.

**Pull request merged**